nmostesting/suites/IS1001Test.py

```diff
@@ -50,8 +50,6 @@ class IS1001Test(GenericTest):
 
     def __init__(self, apis):
         super(IS1001Test, self).__init__(apis)
-        if not ENABLE_HTTPS:

```

Comment:
Could this be reinstated? If it's just being removed for something in our own usage it should probably be done dynamically there.